pytorch_lightning/metrics/classification/accuracy.py

```diff
@@ -37,4 +38,4 @@ def __init__(
         .. deprecated::
             Use :class:`~torchmetrics.Accuracy`. Will be removed in v1.5.0.
         """
-        _ = threshold, top_k, subset_accuracy, compute_on_step, dist_sync_on_step, process_group, dist_sync_fn
+        void(threshold, top_k, subset_accuracy, compute_on_step, dist_sync_on_step, process_group, dist_sync_fn)
```

Comment:
Why do we even have this line?

The docstring acts as `pass`

Reply:
yes, but then IDE complains that you have unused arguments...

Reply:
Does it? I don't get any complain for the stubs in `lightning.py`, for example

<img width="1027" alt="Screenshot 2021-06-02 at 21 04 07" src="https://user-images.githubusercontent.com/11067892/120537901-1f4d8300-c3e6-11eb-86b0-c6502f2e0ae2.png">

Reply:
not sure what you mean here...

Reply:
You said the IDE complains about unused arguments, IIRC you also use PyCharm.

Can you elaborate on those complaints? Where are they?

My screenshot is an example of another function where the arguments are "unused" and the linter doesn't seem to show any warnings.

Reply:
yes, but in your case you have it in docs at least...
one of my cases was running code inspector and there is a group called unused arguments
could be also https://pycodequ.al/docs/pylint-messages/w0613-unused-argument.html ?

Reply:
then why not use `# noqa: W0613` or `# pylint: disable=unused-argument`

instead of introducing `void`